### PR TITLE
Repave clarifications

### PR DESCRIPTION
**Pull request by cloudmaniac**
## Describe the Change

This PR clarifies the different types of repave that can happen in a cluster.

### CODE REVIEW

File: docs/docs-content/clusters/cluster-management/node-pool.md
Comment (antongisli) [reply docs/docs-content/clusters/cluster-management/node-pool.md:25]: do you mean "instance type"? Last sentence can be "Control plane nodes will be repaved sequentially."

Comment (antongisli) [reply docs/docs-content/clusters/cluster-management/node-pool.md:25]: also, avoid word "master", we can use control plane nodes as the term?

**Review by kreeuwijk — APPROVED** [review]
LGTM

**Comment by cloudmaniac** [comment]
@karl-cardenas-coding This can be merged if you're good with the language

File: docs/docs-content/clusters/cluster-management/node-pool.md
Comment (ritawatson) [inline docs/docs-content/clusters/cluster-management/node-pool.md:10]:
```suggestion
A node pool is a group of nodes within a cluster that all have the same configuration. You can use node pools for different workloads. For example, you can create a node pool for your production workloads and another for your development workloads. You can update node pools for active clusters or create a new one for the cluster.
```

Comment (ritawatson) [reply docs/docs-content/clusters/cluster-management/node-pool.md:10]: Reduced the number of occurrences of "node pool".

File: docs/docs-content/clusters/cluster-management/node-pool.md
Comment (ritawatson) [inline docs/docs-content/clusters/cluster-management/node-pool.md:23]:
```suggestion
Different types of repaving operations may occur, depending on what causes them:
```

File: docs/docs-content/clusters/cluster-management/node-pool.md
Comment (ritawatson) [inline docs/docs-content/clusters/cluster-management/node-pool.md:25]:
```suggestion
* **Control plane repave**: This takes place when certain changes are made to the Kubernetes configuration, such as changing the apiServer specification. This type of repave also occurs when there are changes in the hardware specifications of the control plane nodes, such as during a scale-up operation or when changing from one instance type to another. Control plane nodes are repaved sequentially.
```

File: docs/docs-content/clusters/cluster-management/node-pool.md
Comment (ritawatson) [inline docs/docs-content/clusters/cluster-management/node-pool.md:27]:
```suggestion
* **Worker node pool repave**: This happens when changes to a node pool's specifications render the existing nodes incompatible with the pool's criteria. For instance, changing the hardware specifications of a worker pool. Nodes within the affected pool are sequentially replaced with new nodes that meet the updated specifications.
```

File: docs/docs-content/clusters/cluster-management/node-pool.md
Comment (ritawatson) [inline docs/docs-content/clusters/cluster-management/node-pool.md:29]:
```suggestion
* **Full cluster repave**: This occurs if changes are made to the Operating System (OS) layer or if modifications to the Kubernetes layer impact all nodes, such as when upgrading to a different Kubernetes version. All nodes across all pools are sequentially repaved starting with the control plane.
```

File: docs/docs-content/clusters/cluster-management/node-pool.md
Comment (ritawatson) [inline docs/docs-content/clusters/cluster-management/node-pool.md:31]:
```suggestion
You can customize the repave time interval for all node pools except the master pool. The default repave time interval is 0 seconds. You can adjust the node repave time interval during or after cluster creation. If you need to modify the repave time interval post-cluster creation, follow the [Change a Node Pool](#changeanodepool) instructions below.
```

Comment (ritawatson) [reply docs/docs-content/clusters/cluster-management/node-pool.md:31]: Line 31 uses the term "master". In this case, we could say, "You can customize the repave time interval for all node pools except the control plane node pool."

File: docs/docs-content/glossary-all.md
Comment (ritawatson) [inline docs/docs-content/glossary-all.md:161]:
```suggestion
Repavement is the process of replacing a Kubernetes node with a new one. This is typically done when a node is unhealthy or needs to be upgraded. The process involves migrating active workloads to another healthy node, and removing it from the [node pool](/clusters/cluster-management/node-pool/#repave-behavior-and-configuration). This is referred to as draining the node. A new node is created and configured with the same settings as the old node and added back to the pool. The process is fully automated and does not require manual intervention.

```

**Review by ritawatson — APPROVED** [review]
There are some comments to address before this PR can be merged.

**Review by cloudmaniac — COMMENTED** [review]
Accepted suggestions

**Review by karl-cardenas-coding — APPROVED** [review]

**Pull request merged**
